src/server/config/schema.js

```diff
@@ -146,7 +145,9 @@ module.exports = () => Joi.object({
       bounds: Joi.array().items(Joi.array().items(Joi.number()).min(2).required()).min(2)
     }).default()
   }).default(),
-
+  maps: Joi.object({
```

Comment:
I thought it would be easier to document that tilemap is now deprecated and instead you use a new parameter.

also, the manifest is not necessarily tied to "tilemaps", that service array could contain other things down the road as well.